archetypes.js

```diff
@@ -0,0 +1,1484 @@
+  enduranceMax: 100,
+  kickPowerMin: 1,
+  kickPowerMax: 25,
+  kickAccuracyMin: 1,
+  kickaccuracyMax: 25
+};
+
+const safetyAllAround = {
+  position: 'Safety',
+  shortPosition: 'S',
+  archetype: 'All Around',
+  baseTpe: 446,
+  strengthMin: 75,
+  strengthMax: 45,
+  agilityMin: 65,
+  agilityMax: 95,
+  armMin: 1,
+  armMax: 25,
+  intelligenceMin: 45,
+  intelligenceMax: 80,
+  throwingAccuracyMin: 1,
+  throwingAccuracyMax: 25,
+  tacklingMin: 55,
+  tacklingMax: 80,
+  speedMin: 60,
+  speedMax: 95,
+  handsMin: 50,
+  handsMax: 80,
+  passBlockingMin: 1,
+  passBlockingMax: 25,
+  runBlockingMin: 1,
+  runBlockingMax: 25,
+  enduranceMin: 70,
+  enduranceMax: 100,
+  kickPowerMin: 1,
+  kickPowerMax: 25,
+  kickAccuracyMin: 1,
+  kickaccuracyMax: 25
+};
+
+const defensiveTackleNoseTackle = {
+  position: 'Defensive Tackle',
+  shortPosition: 'DT',
+  archetype: 'Nose Tackle',
+  baseTpe: 346,
+  strengthMin: 70,
+  strengthMax: 100,
+  agilityMin: 35,
+  agilityMax: 60,
+  armMin: 1,
+  armMax: 25,
+  intelligenceMin: 45,
+  intelligenceMax: 75,
+  throwingAccuracyMin: 1,
+  throwingAccuracyMax: 25,
+  tacklingMin: 40,
+  tacklingMax: 90,
+  speedMin: 35,
+  speedMax: 70,
+  handsMin: 25,
+  handsMax: 55,
+  passBlockingMin: 1,
+  passBlockingMax: 25,
+  runBlockingMin: 1,
+  runBlockingMax: 25,
+  enduranceMin: 60,
+  enduranceMax: 100,
+  kickPowerMin: 1,
+  kickPowerMax: 25,
+  kickAccuracyMin: 1,
+  kickaccuracyMax: 25
+};
+
+const defensiveTackleRunStuffer = {
+  position: 'Defensive Tackle',
+  shortPosition: 'DT',
+  archetype: 'Run Stuffer',
+  baseTpe: 361,
+  strengthMin: 65,
+  strengthMax: 95,
+  agilityMin: 40,
+  agilityMax: 75,
+  armMin: 1,
+  armMax: 25,
+  intelligenceMin: 45,
+  intelligenceMax: 75,
+  throwingAccuracyMin: 1,
+  throwingAccuracyMax: 25,
+  tacklingMin: 50,
+  tacklingMax: 90,
+  speedMin: 45,
+  speedMax: 75,
+  handsMin: 25,
+  handsMax: 55,
+  passBlockingMin: 1,
+  passBlockingMax: 25,
+  runBlockingMin: 1,
+  runBlockingMax: 25,
+  enduranceMin: 60,
+  enduranceMax: 100,
+  kickPowerMin: 1,
+  kickPowerMax: 25,
+  kickAccuracyMin: 1,
+  kickaccuracyMax: 25
+};
+
+const defensiveTackleInteriorRusher = {
+  position: 'Defensive Tackle',
+  shortPosition: 'DT',
+  archetype: 'Interior Rusher',
+  baseTpe: 346,
+  strengthMin: 55,
+  strengthMax: 90,
+  agilityMin: 50,
+  agilityMax: 80,
+  armMin: 1,
+  armMax: 25,
+  intelligenceMin: 40,
+  intelligenceMax: 70,
+  throwingAccuracyMin: 1,
+  throwingAccuracyMax: 25,
+  tacklingMin: 40,
+  tacklingMax: 90,
+  speedMin: 50,
+  speedMax: 80,
+  handsMin: 30,
+  handsMax: 60,
+  passBlockingMin: 1,
+  passBlockingMax: 25,
+  runBlockingMin: 1,
+  runBlockingMax: 25,
+  enduranceMin: 60,
+  enduranceMax: 100,
+  kickPowerMin: 1,
+  kickPowerMax: 25,
+  kickAccuracyMin: 1,
+  kickaccuracyMax: 25
+};
+
+const defensiveTackleBalanced = {
+  position: 'Defensive Tackle',
+  shortPosition: 'DT',
+  archetype: 'Balanced',
+  baseTpe: 341,
```

Comment:
Should be 361. Looks like you used the stat totals instead of TPE totals on strength and endurance

70+45+1+50+1+45+45+30+1+1+70+1+1 = 361